src/app-layer-htp.c

```diff
@@ -2264,6 +2264,7 @@ static void HTPConfigSetDefaultsPhase1(HTPCfgRec *cfg_prec)
     htp_config_register_response_complete(cfg_prec->cfg, HTPCallbackResponse);
 
     htp_config_set_parse_request_cookies(cfg_prec->cfg, 0);
+    htp_config_set_allow_space_uri(cfg_prec->cfg, 1);
```

Comment:
I think it would be good to check for this in configure and guard it with ifdefs. We did this in the past with HAVE_HTP_URI_NORMALIZE_HOOK (see git grep HAVE_HTP_URI_NORMALIZE_HOOK, although it's now unused and can be removed)

Reply:
Ok, I did not add it to `features` in suricata.c
Should I ?